applications/matter_weather_station/sample.yaml

```diff
@@ -2,8 +2,17 @@ sample:
   name: Matter Weather Station
   description: Matter Weather Station application
 tests:
-  applications.matter_weather_station:
+  applications.matter_weather_station.debug:
     build_only: true
+    extra_args: DCMAKE_BUILD_TYPE=ZDebug CONFIG_CHIP_LIB_SHELL=y
+    platform_allow: thingy53_nrf5340_cpuapp
+    platform_exclude: thingy53_nrf5340_cpuapp_ns
+    integration_platforms:
+      - thingy53_nrf5340_cpuapp
+    tags: ci_build
+  applications.matter_weather_station.release:
+    build_only: true
+    extra_args: DCMAKE_BUILD_TYPE=ZRelease
```

Comment:
Is `DCMAKE_BUILD_TYPE=ZRelease` used at all? Shouldn't we rather use `CONF_FILE=prj_release.conf`?